yarn.lock

```diff
@@ -17,6 +17,14 @@
   dependencies:
     "@babel/highlight" "^7.18.6"
 
+"@babel/code-frame@^7.23.5":
```

Comment:
@raju-opti Can we track down the dep that's using this vulnerable tough-cookie?

Reply:
Maybe this is from `jsdom@^16.4.0`?

Reply:
We can using `yarn why tough-cookie` (it says because "jest#@jest#core#jest-config#jest-environment-jsdom#jsdom"), but updating that is another matter. As you can see, it's long down the path of transitive dependencies. So let's say for example, we update jest, now new `jest` might use new version of `jest-config`, which might have a completely new dpendency currently not in the transitive dep tree.  And this update is in lockfile, so updating packages is not just simple version bump. 

Dependabot can generate fixes for vulnarabilites automaticaly, keeping the package.json and lockfile consistent, and this commit is generated using dependabot. In this case, you can see that there are a lot of new packages in the lockfile.

Also, yarns lockfile makes it especially difficult to manually update versions, cause it's a flat structure. What I propose is we merge this commit, and then fix the new vulnerability using the same method for the new vulnarabilty.